cli/tests/data/package/create-directory/marathon.json.mustache

```diff
@@ -5,8 +5,8 @@
   "cpus": {{cassandra.framework.cpus}},
   "mem": {{cassandra.framework.mem}},
   "uris": [
-    "{{package.assets.uris.cassandra-mesos.tar.gz}}",
-    "{{package.assets.uris.jre-linux.tar.gz}}"
+    "{{resource.assets.uris.cassandra-mesos}}",
+    "{{resource.assets.uris.jre-linux}}"
```

Comment:
We're going to need to change the mustache template to s/./- after resource.assets.uris when we implement `package import`